ui/js/controllers/logviewer.js

```diff
@@ -133,6 +133,9 @@ logViewerApp.controller('LogviewerCtrl', [
                         {label: 'End', value: dateFilter(job.end_timestamp * 1000, thDateFormat)}
                     ];
 
+                    // Test to disable successful steps checkbox on taskcluster jobs
+                    $scope.buildSystemType = job.build_system_type;
```

Comment:
Cool. Ya I wasn't sure if it should be build-system generic or not. Changing now.